x-pack/solutions/search/plugins/enterprise_search/public/applications/enterprise_search_content/components/connectors/connectors.tsx

```diff
@@ -95,6 +97,29 @@ export const Connectors: React.FC<ConnectorsProps> = ({ isCrawler }) => {
             : i18n.translate('xpack.enterpriseSearch.crawlers.title', {
                 defaultMessage: 'Elastic Web Crawler',
               }),
+          description: [
+            <EuiText>
+              <p>
+                <FormattedMessage
+                  id="xpack.enterpriseSearch.webcrawlers.headerContent"
+                  defaultMessage="Discover extract and index searchable content from websites and knowledge bases {learnMoreLink}"
```

Comment:
@JoseLuisGJ Just a punctuation edit if you're touching this copy again soon:

`"Discover, extract, and index searchable content from websites and knowledge bases."`